Sources/Rendering/OpenGL/BufferObject/index.d.ts

```diff
@@ -0,0 +1,87 @@
+import { ObjectType } from './Constants';
+import { vtkAlgorithm, vtkObject } from '../../../interfaces';
+
+/**
+ * Interface for initial values of BufferObject
+ */
+export interface IBufferObjectInitialValues {
+  objectType: ObjectType;
+  context?: WebGLRenderingContext | WebGL2RenderingContext;
+  allocatedGPUMemoryInBytes: number;
```

Comment:
`objectType` and `allocatedGPUMemoryInBytes` should be optional, since generally initial values are optional.